src/amo/components/App/index.js

```diff
@@ -264,10 +250,7 @@ export class AppBase extends React.Component<Props> {
               </div>
             </div>
 
-            <FooterComponent
-              handleViewDesktop={this.onViewDesktop}
-              location={location}
-            />
+            <FooterComponent location={location} />
```

Comment:
I know this isn't something you changed but I am wondering if we could access the location inside the footer component through the `withRouter` HOC? Would that be preferred? Maybe this could be done in a follow up issue..

Reply:
This is a good idea, so I made the change in this patch. It turns out the only reason that `Footer` needed access to `location` was so it could pass it into `LanguagePicker`, so I've updated `LanguagePicker` to use `withRouter`.

Because this is a change from what was reviewed previously, do you mind taking another look, @rebmullin?

Reply:
> This is a good idea, so I made the change in this patch. It turns out the only reason that Footer needed access to location was so it could pass it into LanguagePicker, so I've updated LanguagePicker to use withRouter.

@bobsilverberg, perfect:)

> Because this is a change from what was reviewed previously, do you mind taking another look, @rebmullin?

sure thing, I'll take another look.